config/locales/zh-CN.yml

```diff
@@ -1,9 +1,9 @@
 zh-CN:
   errors:
     messages:
-      not_a_date: "不是一个日期"
-      after: "必须晚于 %{date}"
-      after_or_equal_to: "必须早于或等于 %{date}"
-      before: "必须早于 %{date}"
-      before_or_equal_to: "必须晚于或等于 %{date}"
-      equal_to: "必须等于 %{date}"
+      date_not_a_date: "不是一个日期"
```

Comment:
This one should be `not_a_date`, according to the Enligh locale file:

https://github.com/codegram/date_validator/blob/48b05353e1ab65350d3089ecfce5ce3b0837767a/config/locales/en.yml#L4

Can you change this please? 😄

Reply:
Sorry for the late reply.
fixed. 
Thank you 😄